Docs/src/manual.md

```diff
@@ -885,3 +886,77 @@ One example of this is the Ogre::Light class. It extends AnimableObject and prov
+This class allows you to build a batched object from a series of entities in order to benefit from this behaviour. Batching has implications of it's own though:
+ - Batched geometry cannot be subdivided; that means that the whole group will be displayed, or none of it will. This obivously has culling issues.
+ - A single world transform must apply to the entire batch. Therefore once you have batched things, you can't move them around relative to each other. 
+   That's why this class is most useful when dealing with static geometry (hence the name). 
+   In addition, geometry is effectively duplicated, so if you add 3 entities based on the same mesh in different positions, 
+   they will use 3 times the geometry space than the movable version (which re-uses the same geometry). 
+   So you trade memory and flexibility of movement for pure speed when using this class.
+ - A single material must apply for each batch. In fact this class allows you to use multiple materials, 
+   but you should be aware that internally this means that there is one batch per material. 
+   Therefore you won't gain as much benefit from the batching if you use many different materials; try to keep the number down.
+
+Consult the following links to get more information:
+ - API Documentation: [Ogre::StaticGeometry Class Reference](https://ogrecave.github.io/ogre/api/latest/class_ogre_1_1_static_geometry.html)
+ - Tutorial: [Tutorial - Static Geometry](https://ogrecave.github.io/ogre/api/latest/tut__static_geom.html)
+
+# Instance Manager {#Instance-Manager}
+Instancing is a rendering technique to draw multiple instances of the same mesh using just one render call. There are two kinds of instancing:
+
+Software: Two larges vertex & index buffers are created and the mesh vertices/indices are duplicated N number of times. When rendering, invisible instances receive a transform matrix filled with 0s. This technique can take a lot of VRAM and has limited culling capabilities.
+Hardware: The hardware supports an extra param which allows Ogre to tell the GPU to repeat the drawing of vertices N number of times; thus taking considerably less VRAM. Because N can be controlled at runtime, individual instances can be culled before sending the data to the GPU.
+Hardware techniques are almost always superior to Software techniques, but Software are more compatible, where as Hardware techniques require D3D9 or GL3, and is not supported in GLES2
+
+All instancing techniques require shaders. It is not possible to use instancing with FFP (Fixed Function Pipeline)
+
+Consult the following links to get more information:
+ - API Documentation: [Ogre::InstanceManager Class Reference](https://ogrecave.github.io/ogre/api/latest/class_ogre_1_1_instance_manager.html)
+ - In-depth explanation: [What is instancing?](https://ogrecave.github.io/ogre/api/latest/instancing.html)
+
+# Differences between both (#Differences-between-both)
+<table>
```

Comment:
if you use markdown tables, this could be readable in the source-code as well. See:
https://github.com/OGRECave/ogre/blob/6c376059f03fbcbfd612e7cbf64216cf1b7ebd5b/OgreMain/include/OgreRenderSystem.h#L378

Reply:
Yes, but I thought that the doxygen tables look better.

Anyway, I don't even know how to generate the documentation with doxygen.

I installed it, the CMake detects it, but nothing is generated. It just copies the files. I'm trying to solve this before submitting again.

Reply:
you have to run the "OgreDoc" target. e.g.
```
cmake --build . --target OgreDoc
```

Reply:
Thanks